src/components/buttons/TintButton.tsx

```diff
@@ -26,8 +30,13 @@ const TintButton = ({
       height={`${height}px`}
       justifyContent="center"
       // @ts-expect-error
-      onPress={onPress}
-      style={useMemo(() => ({ overflow: 'hidden' as 'hidden' }), [])}
+      onPress={disabled ? () => undefined : onPress}
+      scale={disabled ? 1 : 0.8}
+      style={useMemo(
+        () => ({ opacity: disabled ? 0.5 : 1, overflow: 'hidden' as 'hidden' }),
+        [disabled]
+      )}
```

Comment:
This is the first time I see this pattern. I wonder if this is not "too much". I cannot justify this

Reply:
Yeah, probably a bit of an over optimisation, I think without `useMemo` is ok.